extensions/coap/tests/CoapIntegrationBase.h

```diff
@@ -45,6 +45,7 @@ class CoapIntegrationBase : public IntegrationBase {
   }
 
   void run(const utils::optional<std::string>& test_file_location = {}, const utils::optional<std::string>& bootstrap_file = {}) override {
+    (void)bootstrap_file;  // against unused variable warnings
```

Comment:
Why don't you leavev the argument unnamed instead?

Reply:
I think I did not realize you can leave the argument unnamed when it has a default value. Updated in [75e1aa](https://github.com/apache/nifi-minifi-cpp/pull/1020/commits/75e1aa33766b2b43b135ebfd2aac312e796d463a)